### ISSUE TITLE
[Request] webP image support

### ISSUE THREAD

**Issue by ignace72** [StigNygaard/xIFr#15]
Hello.
The webP images being more and more present on the internet because the weight gain is impressive.
I was wondering if it was possible to add support for this image format to your xIFr module.
Thank you.

**Comment by StigNygaard** [comment]
Hi @ignace72 
I might add support for other formats. If/When I find some time, I am considering to do a big rewrite of xIFr basing the parsing-part on a new "3rd party" library which supports JPEG, AVIF and PNG. However WEBP is currently _not_ in scope.
I know WEBP currently have pretty widespread use, but in the future I expect [AVIF](https://caniuse.com/avif) (and in the longer run JPEG XL) to be much more important formats on the web.
So in short, I will have to disappoint you. It is unlikely that xIFr ever gets WEBP support.
/Stig

**Comment by ignace72** [comment]
Hello, StigNygaard.
It is true that webP is only a transition because it is based on the vp8 codec which is still from 2008. AVIF is a much more promising solution because it is based on the Av1 codec. For my videos I can achieve a gain of 45% compared to H265 without any visible loss of quality.
I'm not really disappointed because it's not like seeing the exifs quickly is vital for me either, it's just easier to right click than to save the image and open it with The Gimp.
That you are considering AVIF support is already good news for me. For now, it's a bit early because for example my Wordpress gallery manager doesn't support it. I don't know any image host that supports it either.
Thanks to you for your answer.
